project/api/urls.py

```diff
@@ -36,8 +36,6 @@
 urlpatterns = [
     path("", include(ROUTER.urls)),
     path(
-        "mvt-tiles/<int:zoom>/<int:x_cor>/<int:y_cor>",
-        views.mvt_tiles,
-        name="mvt-tiles",
+        "mvt/<int:zoom>/<int:x_cor>/<int:y_cor>/<int:year>", views.mvt_tiles, name="mvt"
```

Comment:
That is not what I meant, actually

Let's keep the previous url `"mvt-tiles/<int:zoom>/<int:x_cor>/<int:y_cor>",`